Library/Formula/proof-general.rb

```diff
@@ -39,6 +40,11 @@ def install
       system "make", "install", *args
 
       man1.install "doc/proofgeneral.1"
+      if build.head?
+          cd "doc" do
```

Comment:
Your indent is too hard here. Looks like you've indented with a tab instead of spaces.